docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -846,7 +827,18 @@ these pieces of data from it. It then performs the following actions:
   
 </Admonition>
 
-## Step 5/5. Remove trust between your clusters
+## Step 5/6. Updating role mappings
```

Comment:
- I would say `Update` instead of `Updating` to match the form of the other steps
- This isn't really a required step in the process, it's more of a note that you can make updates. Should we put it under step 4 instead of making it a separate step? (I think so, and I'd also mention something like "starting version x.y.z  you can update role mappings without creating a new trusted cluster)

Reply:
Will we need the 'starting in version xyz" if the docs are already versioned.
I've added the other changes.